### PR TITLE
Set SDM operator images using toolchain_image_repo var

### PR DESCRIPTION
**Pull request by ChrisSchreiber**

### CODE REVIEW

File: modules/lead/sdm/operator-toolchain-values.tpl [hunk 1]
```diff
@@ -45,6 +51,8 @@ operators:
       value: ${region}
   jenkins:
     enabled: ${operator_jenkins_enabled}
+    image:
+      repository: ${image_repository}
```
Comment (mrparkers) [inline modules/lead/sdm/operator-toolchain-values.tpl]: How can these all share the same repository?  Does that mean the same docker image will be used for each of these?

Comment (mrparkers) [reply modules/lead/sdm/operator-toolchain-values.tpl]: Also the product operator image and converge image are missing

Comment (ChrisSchreiber) [reply modules/lead/sdm/operator-toolchain-values.tpl]: We could make the image repo the same value but I think long term we are going to break each of these into their own chart anyway.

Comment (ChrisSchreiber) [reply modules/lead/sdm/operator-toolchain-values.tpl]: Oops, I misread the first comment but everything should be fixed

**Pull request merged**